src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/BindingStep/config.jelly

```diff
@@ -25,6 +25,7 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <j:set var="expressionAllowed" value="false"/>
```

Comment:
why is an expression not allowed here? or is it this that causes the NPE and you are just hacking around the NPE